ReactWindows/ReactNative/Modules/Storage/AsyncStorageModule.cs

```diff
@@ -20,5 +39,240 @@ public override string Name
                 return "AsyncLocalStorage";
             }
         }
+
+        /// <summary>
+        /// Given an array of keys, this returns a map of (key, value) pairs for the keys found, and
+        /// (key, null) for the keys that haven't been found.
+        /// </summary>
+        /// <param name="keys">Array of key values</param>
+        /// <param name="callback">Callback.</param>
+        [ReactMethod]
+        public void multiGet(JArray keys, ICallback callback)
+        {
+            if (keys == null)
+                throw new ArgumentNullException(nameof(keys));
```

Comment:
I don't think this is the expected behavior.  In Android, if this is called with null keys, then there is a special callback response (i.e., the error is propagated to JS through the callback, rather than by throwing an exception).